src/clients/ntp-client_proposal.rb

```diff
@@ -514,16 +532,34 @@ def dhcp_ntp_items
     #
     # @return [Array<Yast::Term>] ntp address table Item
     def fallback_ntp_items
-      dhcp_items = dhcp_ntp_items
-
-      log.info("Nothing found in /etc/chrony.conf")
-      if dhcp_items.empty?
+      return @cached_fallback_ntp_items if @cached_fallback_ntp_items
+      @cached_fallback_ntp_items = dhcp_ntp_items
+      if !@cached_fallback_ntp_items.empty?
+        log.info("Proposing NTP server list provided by DHCP")
+      else
         log.info("Proposing current timezone-based NTP server list")
-        return timezone_ntp_items
+        @cached_fallback_ntp_items = timezone_ntp_items
       end
+      @cached_fallback_ntp_items
+    end
 
-      log.info("Proposing NTP server list provided by DHCP")
-      dhcp_items
+    # Checking if the user can select one ntp server from the list
+    # of proposed servers.
+    # It does not make sense if there are more than one ntp server has been
```

Comment:
double verb, probably you want something like `It does not make sense if there are more than one ntp server defined`

Reply:
fixed.

Reply:
Fixed.